src/jobs/deploy.yml

```diff
@@ -6,3 +6,12 @@ executor: helm-tools
 steps:
   - checkout
   - deploy
+  - when:
+      condition: <<parameters.send-slack-alert>>
+      steps:
+        - slack/status:
+          context: slack
```

Comment:
need indentation here

Reply:
Also, I'm not sure if we can chain contexts within orbs. I think this should work, but we may want to test it.